generators/php/sdk/src/endpoint/request/WrappedEndpointRequestGenerator.ts

```diff
@@ -110,4 +116,40 @@ export class WrappedEndpointRequestGenerator extends FileGenerator<
             RelativeFilePath.of(this.classReference.name + ".php")
         );
     }
+
+    private generateFieldFromBodyProperty(property: FernIr.InlinedRequestBodyProperty): php.Field | undefined {
+        const type = this.context.phpTypeMapper.convert({ reference: property.valueType, preserveEnums: true });
```

Comment:
`preserveEnums` was previously set to the default value of `false`. This caused request classes to have string fields instead of enums. I overrode this to `true` to align with the [ticket expectations](https://github.com/fern-api/fern/pull/4728/files#diff-5dabf8f86bbbaca12f3e7d1564728dfae7a1b9d7ca78514830a4ef17488ca214R64), but can revert if string is preferred.